locales/ru.json

```diff
@@ -3,40 +3,40 @@
       {
         "title": "Это Iris",
         "heading": "Шейдеры Iris",
-        "subheading": "Взгляд в новую эру графики Minecraft",
+        "subheading": "Взгляд на новую эру графики Minecraft",
```

Comment:
"_Взгляд **в** новую эру графики_" ≈ "_Взгляд **на** новую эру графики_" = "_A look into a new era of graphics_"

A very subtle use of prepositions: "_на_" - when we look at the era itself from the outside. "_в_" - when we delve into the era and explore it personally. Not critical, but it would be better to see an option that more accurately conveys the original idea.

<details>
 <summary>Исходный текст</summary>

 Очень тонкое употребление предлогов: "_на_" - это смотрим на саму эру со стороны. "_в_" - вникаем в эру и исследуем её лично. Не критично, но хотелось бы видеть вариант, более точно передающий исходную мысль.
</details>